RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -163,7 +163,7 @@ public File getOutputBaseDirectory() {
      * @return
      */
     public File getOutputAnalysisBaseDirectory() {
-        return getRuntimeService().getOutputFolderForAnalysis(this);
+        return getRuntimeService().getOutputFolder(this);
```

Comment:
I remember why I did name the methods like this. I tried to mimic the style of a Grails project we developed at the same time.